game/static/game/css/theme.css

```
@@ -197,46 +209,48 @@
 }
 
 
-/* --- GLOBAL NAVBAR STYLES FROM LANDING.CSS --- */
+/* --- GLOBAL ULTRACLEAN RESPONSIVE NAVBAR STYLES --- */
 .navbar {
     display: flex;
     justify-content: space-between;
     align-items: center;
 
     width: calc(100% - 40px);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)